module/zcommon/zfeature_common.c

```diff
@@ -598,6 +598,18 @@ zpool_feature_init(void)
 	zfeature_register(SPA_FEATURE_DRAID,
 	    "org.openzfs:draid", "draid", "Support for distributed spare RAID",
 	    ZFEATURE_FLAG_MOS, ZFEATURE_TYPE_BOOLEAN, NULL);
+
+	{
+	static const spa_feature_t zilsaxattr_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
```

Comment:
[nit] does it really need EXTENSIBLE_DATASET?

Reply:
Doesn't need. would remove it.

Reply:
Uploaded new patch to activate feature on zil_create and deactivate during zil_destroy.  It needs per dataset feature flag "ZFEATURE_FLAG_PER_DATASET", which needs EXTENSIBLE_DATASET. So, would keep it.